documentation/problem-forwarding-to-keptn.md

```diff
@@ -0,0 +1,76 @@
+
+
+## Setting up problem notifications for problems detected on Keptn deployed services
+
+To enable this feature, add the follow Custom Problem Notification to your Dynatrace tenant:
+
+```json
+{
+    "specversion":"1.0",
+    "shkeptncontext":"{PID}",
+    "type":"sh.keptn.events.problem",
+    "source":"dynatrace",
+    "id":"{PID}",
+    "time":"",
+    "contenttype":"application/json",
+    "data": {
+        "State":"{State}",
+        "ProblemID":"{ProblemID}",
+        "PID":"{PID}",
+        "ProblemTitle":"{ProblemTitle}",
+        "ProblemURL":"{ProblemURL}",
+        "ProblemDetails":{ProblemDetailsJSON},
+        "Tags":"{Tags}",
+        "ImpactedEntities":{ImpactedEntities},
+        "ImpactedEntity":"{ImpactedEntity}",
+    }
+}
+```
+
+The dynatrace-service can [configure this automatically in a Dynatrace tenant](auto-tenant-configuration.md#problem-notifications).
+
+The dynatrace-service will parse the `Tags` field and use the `keptn_project`, `keptn_service` and `keptn_stage` tags that come directly from the impacted entities to ensure that the event is mapped to the correct project service and stage.
+
+*Note:* Dynatrace Alerting Profiles that include problems on services without Keptn tags cannot be mapped through this capability.
+
+
+## Setting up problem notifications for any type of detected problem, e.g., infrastructure
+
+To send any type of problem to Keptn and orchestrate auto-remediation workflows, simply specify the project, service and stage as part of the data structure that is sent to Keptn. For example, the following Dynatrace custom notification integration payload will send all problems to a Keptn project called `dynatrace`, stage called `production` and service called `allproblems`:
+
+```json
+{
+    "specversion":"1.0",
+    "shkeptncontext":"{PID}",
+    "type":"sh.keptn.events.problem",
+    "source":"dynatrace",
+    "id":"{PID}",
+    "time":"",
+    "contenttype":"application/json",
+    "data": {
+        "State":"{State}",
+        "ProblemID":"{ProblemID}",
+        "PID":"{PID}",
+        "ProblemTitle":"{ProblemTitle}",
+        "ProblemURL":"{ProblemURL}",
+        "ProblemDetails":{ProblemDetailsJSON},
+        "Tags":"{Tags}",
+        "ImpactedEntities":{ImpactedEntities},
+        "ImpactedEntity":"{ImpactedEntity}",
+        "KeptnProject" : "dynatrace",
+        "KeptnService" : "allproblems",
+        "KeptnStage" : "production"
+    }
+}
+```
+
+When the dynatrace-service receives this `sh.keptn.events.problem` event it will parse the fields `KeptnProject`, `KeptnService` and `KeptnStage` and will then send a `sh.keptn.event.problem.open` event to Keptn including the rest of the problem details. This allows you to send any type of Dynatrace detected problem to Keptn and let Keptn execute a remediation workflow.
```

Comment:
... will then send a `sh.keptn.event.[STAGE_NAME].remediation.triggered` event to Keptn.